devpack/src/main/java/io/neow3j/devpack/Hash160.java

```diff
@@ -32,6 +34,11 @@
 
     /**
      * Checks if this {@code Hash160} is valid, i.e. is 20 bytes long.
+     * <p>
+     * This method is useful when a contract method takes a {@code Hash160} as a parameter and
+     * you want to make sure that the underlying value really is a valid hash. The contract does
+     * not include or enforce this check automatically, because that would consume extra GAS even
+     * if you don't require that check.
```

Comment:
To be congruent with the same method for `Hash256`:

```suggestion
     * This method is useful when a contract method takes a {@code Hash160} as a parameter and
     * you want to make sure that the underlying value really is a valid hash. The compiler and
     * NeoVM don't include or enforce this check automatically, because that would consume extra
     * GAS even if you don't require that check.
```

Reply:
Good catch!